src/libmongoc/tests/unified/entity-map.c

```diff
@@ -779,8 +669,10 @@ entity_client_new (entity_map_t *em, bson_t *bson, bson_error_t *error)
    mongoc_client_set_error_api (client, MONGOC_ERROR_API_VERSION_2);
    entity->value = client;
    mongoc_client_set_apm_callbacks (client, callbacks, entity);
+   mongoc_client_set_structured_log_opts (client, log_opts);
 
    if (can_reduce_heartbeat && em->reduced_heartbeat) {
+      // @todo Examine whether this is needed in addition to the URI param above
```

Comment:
`min_heartbeat_frequency_msec` is [checked for single-threaded scanning](https://github.com/mongodb/mongo-c-driver/blob/9ed337b634ef18089bfa02390b6c662417b87382/src/libmongoc/src/mongoc/mongoc-topology.c#L1180). I expect this enables reducing the heartbeat attempt frequency below 500ms. 500ms is the [non-configurable minimum](https://github.com/mongodb/specifications/blob/787bbe67bdffb614270654372112e93fc2cd3eb3/source/server-discovery-and-monitoring/server-monitoring.md#minheartbeatfrequencyms) in the spec.

Reply:
Ah, that makes sense. Thanks for the explanation, I'll update the comment.